### ISSUE TITLE
Add github action to auto label issues

### ISSUE THREAD

**Issue by muthuannamalai12** [DhairyaBahl/React-Messenger-App#116]
Add GitHub action to auto label issues with gssoc21 label when contributor asks to work under gssoc21.

**Comment by DhairyaBahl** [comment]
@muthuannamalai12 You can work on this issue but keep me posted on what you are doing ^_^

All the best :)

**Comment by muthuannamalai12** [comment]
@DhairyaBahl Could you please add the gssoc21 label and level label??

**Comment by DhairyaBahl** [comment]
@muthuannamalai12 Labels Added ! Sorry, I didn't know that you are a gssoc student ! All the best ^_^

**Comment by Ankita297** [comment]
Can i work on this @DhairyaBahl ?

**Comment by DhairyaBahl** [comment]
@Ankita297 This issue is already resolved ig ! ```gssoc'21``` label is added when someone uses the word gssoc21

**Comment by Ankita297** [comment]
https://user-images.githubusercontent.com/67833077/116282864-e861cd00-a7a8-11eb-9a7c-1c1a1661abc6.png

@DhairyaBahl is this okay ?

**Comment by DhairyaBahl** [comment]
@Ankita297 Yes its okay but the label is gssoc21 in lowercase ! So kindly make the required changes and create a PR for the same ^_^

**Comment by subhangi2731** [comment]
@abhijeet007rocks8 please assign this issue to me

**Comment by Ankita297** [comment]
@subhangi2731 the issue is already resolved .
